packages/pangea-node-sdk/src/services/authn/user/invites.ts

```diff
@@ -12,23 +12,46 @@ export default class UserInvites extends BaseService {
   // authn::/v1/user/invite/list
   /**
    * @summary List invites
-   * @description Lookup active invites for the userpool
+   * @description Look up active invites for the userpool.
+   * @operationId authn_post_v1_user_invite_list
+   * @param {Object} request - Supported options:
+   *   - filter (object)
+   *   - last (string): Reflected value from a previous response to
+   * obtain the next page of results.
+   *   - order (AuthN.ItemOrder): Order results asc(ending) or desc(ending).
+   *   - order_by (AuthN.User.ListOrderBy): Which field to order results by.
+   *   - size (number): Maximum results to include in the response.
    * @returns {Promise<PangeaResponse<AuthN.User.Invite.ListResult>>} - A list of pending user invitations
    * @example
-   * const response = await authn.user.invites.list();
+   * ```js
+   * const response = await authn.user.invites.list(
+   *   {
+   *     order: AuthN.ItemOrder.ASC,
+   *     order_by: AuthN.User.ListOrderBy.ID,
+   *     size: 20,
+   *   }
+   * );
+   * ```
    */
-  list(): Promise<PangeaResponse<AuthN.User.Invite.ListResult>> {
-    return this.post("user/invite/list", {});
+  list(request?: AuthN.User.ListRequest): Promise<PangeaResponse<AuthN.User.Invite.ListResult>> {
```

Comment:
We should do another interface in AuthN.User.Invites.ListRequest due to this one does not have use_new as User.ListRequest